.github/workflows/build-package.yml

```diff
@@ -0,0 +1,19 @@
+name: Debian Package Build
+on:
+  pull_request:
+    branches:
+      - current
```

Comment:
There should be only sagitta
Why do you need build the package for the  current/equuleus PR’s?

Reply:
Done